src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -106,9 +106,11 @@ const ForgotPassword = ({ intl, location, dependencies: { dopplerLegacyClient }
         <h5>{_('login.forgot_password')}</h5>
         <p className="content-subtitle">{_('forgot_password.description')}</p>
         {sentEmail ? (
-          <div className="forgot-message bounceIn">
-            <FormattedHTMLMessage tagName="div" id="forgot_password.confirmation_message_HTML" />
-            <LinkToLoginSuccess />
+          <div className="forgot-message">
```

Comment:
Why do we have a _forgot-message_ class? it does not look as something reusable.

Anyway, this was already here, so fixing it is out of the scope of this PR.

Reply:
We use this class within the web application so that when you clean the form, keep the height and do not lift your foot sharply. It is something that we could eliminate and look for a better way that does not happen.